Frameworks/CoreText/CTFramesetter.mm

```diff
@@ -32,7 +32,7 @@ @implementation _CTFramesetter : NSObject
     // Call _DWriteWrapper to get _CTLine object list that makes up this frame
     _CTTypesetter* typesetter = static_cast<_CTTypesetter*>(framesetter->_typesetter);
     if (range.length == 0L) {
-        range.length = typesetter->_characters.size() - range.location;
+        range.length = [typesetter->_string length] - range.location;
```

Comment:
it's stored as an NSString :(

---
In reply to: [92743322](https://github.com/Microsoft/WinObjC/pull/1558#discussion_r92743322) [](ancestors = 92743322)